src/dataLoaders/components/collectorsWizard/CollectorsStepSwitcher.tsx

```diff
@@ -6,33 +6,43 @@ import SelectCollectorsStep from 'src/dataLoaders/components/collectorsWizard/se
 import SelectCollectorsStep2 from 'src/dataLoaders/components/collectorsWizard/select/SelectCollectorsStep2'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import PluginConfigSwitcher from 'src/dataLoaders/components/collectorsWizard/configure/PluginConfigSwitcher'
+import PluginCreateConfigurationWizard from 'src/dataLoaders/components/collectorsWizard/verify/PluginConfigSwitcher'
+
 import VerifyCollectorsStep from 'src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep'
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
 // Types
 import {CollectorsStep} from 'src/types/dataLoaders'
-import {CollectorsStepProps} from 'src/dataLoaders/components/collectorsWizard/CollectorsWizard'
+import {PluginCreateConfigurationStepProps} from 'src/writeData/components/PluginCreateConfigurationWizard'
 
 interface Props {
-  stepProps: CollectorsStepProps
+  // stepProps: CollectorsStepProps
+  stepProps: PluginCreateConfigurationStepProps
 }
 
 @ErrorHandling
 class StepSwitcher extends PureComponent<Props> {
   public render() {
     const {stepProps} = this.props
     let selector
+    let configure
     if (isFlagEnabled('telegrafUiRefresh')) {
       selector = <SelectCollectorsStep2 {...stepProps} />
     } else {
       selector = <SelectCollectorsStep {...stepProps} />
     }
 
+    if (isFlagEnabled('telegrafUiRefresh')) {
+      configure = <PluginCreateConfigurationWizard />
+    } else {
+      configure = <PluginConfigSwitcher />
+    }
```

Comment:
there's already a condition for

```ts
if (isFlagEnabled('telegrafUiRefresh')) {
...
```
it probably makes more sense to use that existing conditional statement

Reply:
good point, went ahead and condensed these